coordinator/src/main/scala/com/socrata/datacoordinator/service/Main.scala

```diff
@@ -253,52 +259,11 @@ object Main {
 
         val serv = new Service(serviceConfig, operations.processMutation, operations.processCreation, getSchema _, getRollups,
           operations.exporter, secondaries, operations.datasetsInStore, operations.versionInStore,
-          operations.ensureInSecondary, operations.ensureInSecondaryGroup, operations.secondariesOfDataset, operations.listDatasets, operations.deleteDataset,
+          operations.ensureInSecondary, operations.ensureInSecondaryGroup, operations.secondariesOfDataset, operations.listDatasets, operations.deleteDataset, operations.removeDataset,
           serviceConfig.commandReadLimit, common.internalNameFromDatasetId, common.datasetIdFromInternalName, operations.makeReportTemporaryFile)
 
         val finished = new CountDownLatch(1)
-        val tableDropper = new Thread() {
-          setName("table dropper")
-          override def run() {
-            do {
-              try {
-                for(u <- common.universe) {
-                  while(finished.getCount > 0 && u.tableCleanup.cleanupPendingDrops()) {
-                    u.commit()
-                  }
-                }
-              } catch {
-                case e: Exception =>
-                  log.error("Unexpected error while dropping tables", e)
-              }
-            } while(!finished.await(30, TimeUnit.SECONDS))
-          }
-        }
-
-        val logTableCleanup = new Thread() {
```

Comment:
I don't think you want to delete the logTableCleanup thread. It does something completely different to the table dropper.